{"uuid": "d60b06cd-e7e2-48f9-97e4-359255bdcc0c", "children": ["9d8b3392-6bcf-4df8-8105-67152028baf2"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 29.024\n  (Session info: chrome=138.0.7204.169)\nStacktrace:\n0   chromedriver                        0x0000000103128ea4 cxxbridge1$str$ptr + 2722840\n1   chromedriver                        0x0000000103120dac cxxbridge1$str$ptr + 2689824\n2   chromedriver                        0x0000000102c723ec cxxbridge1$string$len + 90648\n3   chromedriver                        0x0000000102c5c12c chromedriver + 196908\n4   chromedriver                        0x0000000102c7e378 cxxbridge1$string$len + 139684\n5   chromedriver                        0x0000000102cfabf0 cxxbridge1$string$len + 649756\n6   chromedriver                        0x0000000102cad834 cxxbridge1$string$len + 333408\n7   chromedriver                        0x00000001030ebfc0 cxxbridge1$str$ptr + 2473268\n8   chromedriver                        0x00000001030ef22c cxxbridge1$str$ptr + 2486176\n9   chromedriver                        0x00000001030cda08 cxxbridge1$str$ptr + 2348924\n10  chromedriver                        0x00000001030efae8 cxxbridge1$str$ptr + 2488412\n11  chromedriver                        0x00000001030bea98 cxxbridge1$str$ptr + 2287628\n12  chromedriver                        0x000000010310f9d8 cxxbridge1$str$ptr + 2619212\n13  chromedriver                        0x000000010310fb64 cxxbridge1$str$ptr + 2619608\n14  chromedriver                        0x00000001031209e8 cxxbridge1$str$ptr + 2688860\n15  libsystem_pthread.dylib             0x00000001967c5f94 _pthread_start + 136\n16  libsystem_pthread.dylib             0x00000001967c0d34 thread_start + 8\n\n", "trace": "  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/conftest.py\", line 74, in driver\n    driver.get(config[\"url\"])\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 473, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 448, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1753698720212, "stop": 1753698752101}], "start": 1753698720212, "stop": 1753698752213}